newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -20,6 +24,18 @@ type State = {
   paused: boolean,
 };
 
+const styles = {
+  timeField: {
+    width: 75,
```

Comment:
Having the width is fine. I think we could remove the paddingTop and the bottom

Reply:
no need for it with the `<Line noMargin alignItems="center">`